src/Datadog.Trace.AspNet/TracingHttpModule.cs

```diff
@@ -161,6 +160,18 @@ private void OnEndRequest(object sender, EventArgs eventArgs)
                     app.Context.Items[_httpContextScopeKey] is Scope scope)
                 {
                     scope.Span.SetHttpStatusCode(app.Context.Response.StatusCode, isServer: true);
+
+                    if (app.Context.Items["__Datadog.Trace.ClrProfiler.Managed.AspNetMvcIntegration-aspnet.resourcename"] is string resourceName
```

Comment:
nit: can we declare the key as a constant somewhere else?

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1288/commits/461b9a1ddf7fe83dd31bcd399a4721a42bf83391